test/functional/IllegalAccessError_for_protected_method/playlist.xml

```diff
@@ -22,7 +22,27 @@
 -->
 <playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TKG/playlist.xsd">
 	<test>
-		<testCaseName>IllegalAccessProtectedMethodTest</testCaseName>
+		<testCaseName>IllegalAccessProtectedMethodTest_hs</testCaseName>
+		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) \
+	-javaagent:$(Q)$(TEST_RESROOT)$(D)IllegalAccessProtectedMethod.jar$(Q) \
+	-cp $(Q)$(RESOURCES_DIR)$(P)$(TESTNG)$(P)$(LIB_DIR)$(D)asm-all.jar$(P)$(TEST_RESROOT)$(D)IllegalAccessProtectedMethod.jar$(Q) \
+	org.testng.TestNG -d $(REPORTDIR) $(Q)$(TEST_RESROOT)$(D)testng.xml$(Q) \
+	-testnames IllegalAccessProtectedMethodTest \
+	-groups $(TEST_GROUP) \
+	-excludegroups $(DEFAULT_EXCLUDE); \
+	$(TEST_STATUS)</command>
+		<levels>
+			<level>extended</level>
+		</levels>
+		<groups>
+			<group>functional</group>
+		</groups>
+		<impls>
+			<impl>hotspot</impl>
```

Comment:
Do we need to support a hotspot-only test playlist? Adding `openj9|ibm` `impls` is probably sufficient to avoid hotspot test failures when they consume OpenJ9-only test suites.

Reply:
Yes it will avoid the failure, but this test was running and passing on hotspot before. The point is to run the tests, not exclude them.

Reply:
On Adoptium jenkins, this test is running and passed for hotspot [link](https://ci.adoptium.net/job/Test_openjdk17_hs_extended.functional_x86-64_linux/419/tapTestReport/aqa-tests%2FTKG%2Foutput_17180643782135%2FTest_openjdk17_hs_extended.functional_x86-64_linux.tap-1/). So it would be better to keep it for hotspot too.